File: ssaExample/types_Checker.java
import java.util.Map;

public class types_Checker {

	public types_Config conf;
	public types_Context ctxt;
	public token_FileSet fset;
	public types_Package pkg;
	public types_Info Info;
	public types_version version;
	public Map<Object, types_declInfo> objMap;
	public Map<types_importKey, types_Package> impMap;
	public types_instanceLookup valids;
	public Map<String, Map<String, Boolean>> pkgPathMap;
	public Map<types_Package, Boolean> seenPkgMap;
	public ast_File[] files;
	public Map<token_File, types_version> posVers;
	public types_PkgName[] imports;
	public Map<types_dotImportKey, types_PkgName> dotImportMap;
	public Map<ast_Ident, types_TypeParam> recvTParamMap;
	public Map<types_TypeName, Boolean> brokenAliases;
	public Map<types_Union, types__TypeSet> unionTypeSets;
	public types_monoGraph mono;
	public Object firstErr;
	public Map<types_TypeName, types_Func[]> methods;
	public Map<Object, types_exprInfo> untyped;
	public types_action[] delayed;
	public Object[] objPath;
	public Object[] cleaners;
	public types_environment environment;
	public Integer indent;
}

File: ssaExample/types_Checker.kt
class types_Checker {

	var conf: types_Config? = null
	var ctxt: types_Context? = null
	var fset: token_FileSet? = null
	var pkg: types_Package? = null
	var Info: types_Info? = null
	var version: types_version? = null
	var objMap: Map<Any, types_declInfo>? = null
	var impMap: Map<types_importKey, types_Package>? = null
	var valids: types_instanceLookup? = null
	var pkgPathMap: Map<String, Map<String, Boolean>>? = null
	var seenPkgMap: Map<types_Package, Boolean>? = null
	var files: Array<ast_File>? = null
	var posVers: Map<token_File, types_version>? = null
	var imports: Array<types_PkgName>? = null
	var dotImportMap: Map<types_dotImportKey, types_PkgName>? = null
	var recvTParamMap: Map<ast_Ident, types_TypeParam>? = null
	var brokenAliases: Map<types_TypeName, Boolean>? = null
	var unionTypeSets: Map<types_Union, types__TypeSet>? = null
	var mono: types_monoGraph? = null
	var firstErr: Any? = null
	var methods: Map<types_TypeName, Array<types_Func>>? = null
	var untyped: Map<Any, types_exprInfo>? = null
	var delayed: Array<types_action>? = null
	var objPath: Array<Any>? = null
	var cleaners: Array<Any>? = null
	var environment: types_environment? = null
	var indent: Int? = null
}
